manifests/daemon.pp

```diff
@@ -67,6 +67,7 @@
   Hash[String, Scalar] $env_vars          = {},
   Optional[String] $env_file_path         = $prometheus::env_file_path,
   Optional[String[1]] $extract_command    = $prometheus::extract_command,
+  Stdlib::Absolutepath $achive_bin_path   = "/opt/${name}-${version}.${os}-${arch}/${name}",
```

Comment:
should this be archive_bin_path, and not achive..? Also please add a documentation for the param to the file.